src/OrchardCore.Modules/OrchardCore.Sitemaps/Controllers/SitemapController.cs

```diff
@@ -86,7 +86,7 @@ public async Task<IActionResult> Index(string sitemapId, CancellationToken cance
 
                     document.Declaration = new XDeclaration("1.0", "utf-8", null);
 
-                    using var stream = MemoryStreamFactory.GetStream();
+                    var stream = MemoryStreamFactory.GetStream();
```

Comment:
Is this actually good to keep this stream around? Or should it be a normal `MemoryStream` isntead?

Reply:
I assume it gets disposed after File(stream) respond to the user